mlir/include/mlir/Dialect/MPI/IR/MPIBase.td

```diff
@@ -0,0 +1,39 @@
+//===- MPIBase.td - Base defs for mpi dialect --*- tablegen -*-==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MPI_BASE
+#define MPI_BASE
+
+include "mlir/IR/OpBase.td"
+
+def MPI_Dialect : Dialect {
+  let name = "mpi";
+  let cppNamespace = "::mlir::mpi";
+  let description = [{
+    This dialect models the Message Passing Interface (MPI), version 4.0. It is meant
+    to serve as a targetable dialect, that can be lowered to multiple MPI implementations
+    and hide differences in ABI. The dialect models the functions of the MPI
```

Comment:
The second sentence hear could be more assertive, IMO. The dialect is not only meant this way and can't only do these things, but it IS designed liked this and it DOES this and this.